### ISSUE TITLE
TS-KSR branches

### ISSUE THREAD

**Issue by mrand** [Judo-Documentation-Project/budotree#27]
Per https://en.wikipedia.org/wiki/Tenshin_Sh%C5%8Dden_Katori_Shint%C5%8D-ry%C5%AB and other sources, there are multiple branches of KSR. I looked quickly and couldn't immediately nail down a common ancestor.  That page lists four of them:

- Ōtake
- Sugino Yoshio (whom you have an entry for)
- Hatakeyama
- Sugawara Budo

Reasonably certain Otake in that list refers to [Otake Risuke](https://en.wikipedia.org/wiki/Risuke_Otake) (who passed on his lineage to his older son Otake Nobutoshi)

I am not sure what to make of [this](https://ja.wikipedia.org/wiki/%E5%A4%A9%E7%9C%9F%E6%AD%A3%E4%BC%9D%E9%A6%99%E5%8F%96%E7%A5%9E%E9%81%93%E6%B5%81) linage: which appears to list Otake Nobutoshi under [Otake Toshinori](https://ja.wikipedia.org/wiki/%E5%A4%A7%E7%AB%B9%E5%88%A9%E5%85%B8), which seems unlikely to me.

More on Risuke:
http://katorishintoryu-usa.org/Otake-Sensei.html
https://katorivietnam.org/katori-shinto-ryu-sensei-details/

**Comment by fsmunoz** [comment]
Indeed, TSKSR has many lines. I added the Sugino one almost as a reply to a challenge made in jest some months ago, that with the amount of arts Sugino knew it would stress the limits of the approach (I think it worked perfectly fine).

I need to go back to what I discovered (one of the problems of moving around in the tree is that I can remember that some months ago I knew a lot about TSKSR lineage history, branches, etc, because I was reading all the different resources, but I have now mostly forgot about it). TSKSR is to me in high priority to add, so I will follow up on this as soon as I'm done with the other ones, and comment here.

**Comment by fsmunoz** [comment]
@mrand thank you for your feedback! I will work on Draeger in the next weeks based on that information and ping you back; as often happens, it will not be a 100% coverage since many paths open up and one must stop at a certain point, but it will at least provide a good starting point.

**Comment by fsmunoz** [comment]
An interesting interview here, commenting to come back to it later: https://www.youtube.com/watch?v=_qqVcmgcR7k ("Hoplology: What Donn Draeger taught, and how you get started by Liam Keeley")

**Comment by fsmunoz** [comment]
@mrand I've added Draeger and the TSKSR branch:

![image](https://github.com/user-attachments/assets/bb7d1a55-7f8d-4db3-b24e-5373d579de8e)

As always, it's not "complete", since each individual can be investigated deeper, and on top of that there are branches that aren't there yet. This includes eg. Muso-ryu for Draeger plus other connections, the initial goal was to add Draeger and provide a good basis for further work.

The link to the existing tree is through [Shiina Ichizo](https://budotree.judoc.org/tree.html?id=JDP-35); the others mentioned as Sugino's teacher are likely also similarly linked, but I think there is some confusion with Japanese names that must be carefully investigated to determine some aspects of it.

I've used your link to start with:
![image](https://github.com/user-attachments/assets/ab7af779-4143-42ab-9ebc-3f6df9f155c2)

... and found https://lifeforasword.wordpress.com/2013/11/14/exkurs-die-moderne-geschichte-der-katori-shinto-ryu/ (which is the original AFAIK of the several translation out there) to assist with the rest, plus the official pages of some of the present-day dojos:
![image](https://github.com/user-attachments/assets/c7c2f132-4d36-4a70-bf3f-cec63ce9b32d)


I'll take a look at the non-TSKSR teachers you mentioned as well for a next batch of investigation and changes.
